java/runtime/src/main/resources/ray.default.conf

```diff
@@ -31,11 +31,20 @@ ray {
     num-java-workers-per-process: 1
     /// The jvm options for java workers of the job.
     jvm-options: []
-    // Environment variables to be set on worker processes.
-    worker-env {
-      // key1 : "value1"
-      // key2 : "value2"
+
+    runtime-env: {
```

Comment:
Can we move `runtime-env` out from `job` field? It is not in job config in Python API.

Reply:
Runtime env is set for job.  This is the way we align it to job config in python.

Reply:
If you have any concern on this structure, you could file an issue for that, and then do more discussions. This PR is not aiming to change the original structure.